tests/experimental/datasources/test_postgres_datasource.py

```diff
@@ -496,8 +496,7 @@ def test_sort_batch_list_by_unknown_key(create_source):
 @pytest.mark.unit
 def test_data_source_json_has_properties(create_source):
     with create_source(lambda _: None) as source:
-        assert (
-            type(TableAsset.order_by) == property,
+        assert type(TableAsset.order_by) == property, (
```

Comment:
The changes in this file aren't directly related to my change. They are testing errors I uncovered while making this change and rolled them in. These test asserts always succeed since `assert (test, message)` always passes, since it is asserting a tuple, which is distinct from `assert test, (message)`.